tests/wdio.dev.conf.ts

```diff
@@ -0,0 +1,11 @@
+// wdio.dev.conf.ts
+// extends te main configuration file for the development environment (make dev)
+// it will connect to the webpack-dev-server running locally on port 8080
+import { deepmerge } from 'deepmerge-ts';
```

Comment:
I assume we somehow already have this dependency? It's not listed as a direct dependency, which it probably should. Or, use lodash's merge?

Reply:
ok, I will give it a shot. This was copied from webdriverio site. 
https://webdriver.io/docs/organizingsuites#inherit-from-main-config-file